Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1824,7 +1887,7 @@ void GridTrackSizingAlgorithm::run()
 
     // Step 2.
     if (!m_contentSizedTracksIndex.isEmpty())
-        resolveIntrinsicTrackSizes();
+        (m_renderGrid->isMasonry()) ? resolveIntrinsicTrackSizesMasonry() : resolveIntrinsicTrackSizes();
```

Comment:
Since the non-masonry case is likely to be more common as of now I think it would be better to flip these